docs/docsite/conf.py

```diff
@@ -8,10 +8,10 @@
 sys.path.insert(0, os.path.abspath('./rst/rest_api/_swagger'))
 
 project = u'Ansible AWX'
-copyright = u'2023, Red Hat'
+copyright = u'2024, Red Hat'
 author = u'Red Hat'
 
-pubdateshort = '2023-08-04'
+pubdateshort = '2024-11-22'
 pubdate = datetime.strptime(pubdateshort, '%Y-%m-%d').strftime('%B %d, %Y')
```

Comment:
you could experiment with removing this pubdate info and pubdateshort with something like [this](https://github.com/ansible/ansible-documentation/blob/devel/docs/docsite/rst/conf.py#L288) which we use on the core/package docs... and ... never update it again :-)